### PR TITLE
JBPM-5910: Data source management initialization should retry

### PR DESCRIPTION
**Pull request by wmedvede**

### CODE REVIEW

**Comment by wmedvede** [comment]
Hello @pefernan , would you mind review this PR?
Thanks!

**Comment by pefernan** [comment]
@wmedvede just a minor thing... I see that you are adding whitespaces on your parenthesis. Are you using the right code-style sheet?

Other than that +1

**Comment by wmedvede** [comment]
@pefernan I didn't know there was changes in the code-style sheet...
I was downloaded and applied.
Thanks!

**Comment by pefernan** [comment]
+1 when green

File: kie-wb-common-screens/kie-wb-common-datasource-mgmt/kie-wb-common-datasource-mgmt-backend/src/main/java/org/kie/workbench/common/screens/datasource/management/backend/DataSourceManagementBootstrap.java [hunk 1]
```diff
@@ -34,71 +37,161 @@
...
+
+    protected void initializeConfigParams() {
+        String value = null;
+        try {
+            value = getManagedProperty(DataSourceSettings.getInstance().getProperties(),
+                                       DEPLOYMENTS_INITIALIZATION_RETRIES,
+                                       Long.toString(deploymentsInitializationRetries));
+            deploymentsInitializationRetries = Long.parseLong(value);
+        } catch (NumberFormatException e) {
+            logger.warn("Wrong integer value: " + value + " was set for property: " + DEPLOYMENTS_INITIALIZATION_RETRIES);
```
Comment (jhrcek) [inline kie-wb-common-screens/kie-wb-common-datasource-mgmt/kie-wb-common-datasource-mgmt-backend/src/main/java/org/kie/workbench/common/screens/datasource/management/backend/DataSourceManagementBootstrap.java]: This looks suspicious. Maybe I misunderstand the code, but doesn't this leave the deploymentsInitializationRetries uninitialized in cases when NumberFormatException is thrown?

Comment (wmedvede) [inline kie-wb-common-screens/kie-wb-common-datasource-mgmt/kie-wb-common-datasource-mgmt-backend/src/main/java/org/kie/workbench/common/screens/datasource/management/backend/DataSourceManagementBootstrap.java]: not really, see above that both parameters are already initialized when defined.
    protected long deploymentsInitializationRetries = 20;
    protected long deploymentsInitializationDelay = 30000;
so if a parse exception is produced the original values are preseved.

Comment (wmedvede) [inline kie-wb-common-screens/kie-wb-common-datasource-mgmt/kie-wb-common-datasource-mgmt-backend/src/main/java/org/kie/workbench/common/screens/datasource/management/backend/DataSourceManagementBootstrap.java]: oh, I see. So the missing piece is " Using default value XXX"

Comment (wmedvede) [inline kie-wb-common-screens/kie-wb-common-datasource-mgmt/kie-wb-common-datasource-mgmt-backend/src/main/java/org/kie/workbench/common/screens/datasource/management/backend/DataSourceManagementBootstrap.java]: Fixed

Comment (jhrcek) [inline kie-wb-common-screens/kie-wb-common-datasource-mgmt/kie-wb-common-datasource-mgmt-backend/src/main/java/org/kie/workbench/common/screens/datasource/management/backend/DataSourceManagementBootstrap.java]: Thanks :) +1

**Pull request merged**